week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -37,42 +36,94 @@
  * The input of this function is a simple Array of Strings, look for the concrete example in the code.
- * 
+ *
  * All of your HTML elements should go inside the Div tag with the id "content".
  *
  * Hint for type of lists in HTML: https://www.w3schools.com/html/html_lists.asp
  */
 function insertShoppingList(shoppingList) {
-  //Write your code in here
+  //Write your code in here'
+  let contentList = document.querySelector("#content");
+  var newIl = document.createElement("ul");
+  for (let i = 0; i < shoppingList.length; i++) {
+    var li = document.createElement("li");
+    li.textContent = shoppingList[i];
+    newIl.appendChild(li);
+  }
+  contentList.appendChild(newIl);
 }
 
 /**
  * I'd like to display my three favorite books inside a nice webpage!
- * 
+ *
  * Iterate through the array of books passed into this function and insert each book to page as follows:
  *   - Create a <ul> list and display each book details in <li> element.
  *   - For each book, create a <p> element with the book title and author and append it to the page.
  *   - Add an <img> after <p> element to each book that links to a URL of the book cover.
  *     You should find an appropriate image to each book.
  *   - Change the style of the book depending on whether you have read it (green) or not (red).*
- * 
+ *
  * Find in the code what properties a book object has in the array.
- *  
+ *
  * All of your HTML elements should go inside the Div tag with the id "content".
- * 
+ *
  * The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com
  */
 function insertBooks(books) {
   //Write your code in here
+  books[0].imgUrl =
+    "https://images-na.ssl-images-amazon.com/images/I/410RTQezHYL._SX326_BO1,204,203,200_.jpg";
+  books[1].imgUrl =
+    "https://images-na.ssl-images-amazon.com/images/I/71HMyqG6MRL.jpg";
+  books[2].imgUrl =
+    "https://images-na.ssl-images-amazon.com/images/I/418M2053aNL.jpg";
+  let titulo = document.querySelector("#content");
+  let createH1 = document.createElement("h1");
+  createH1.textContent = "Book List";
+  titulo.appendChild(createH1);
+
+  let info = document.querySelector("#content");
+
+  var ulEl = document.createElement("ul");
+
+  for (let i = 0; i < books.length; i++) {
+    var liEl = document.createElement("li");
+    var paragraph = document.createElement("p");
+    paragraph.textContent = books[i].title + " - " + books[i].author;
+    liEl.appendChild(paragraph);
+    var image = document.createElement("img");
+    image.src = books[i].imgUrl;
+    liEl.appendChild(image);
+    ulEl.appendChild(liEl);
+    liEl.style.width = "400";
+    image.style.width = "200";
+    liEl.style.listStyleType = "none";
+    liEl.style.margin = "20";
+    if (books[i].alreadyRead === true) {
+      liEl.style.backgroundColor = "green";
+    } else {
+      liEl.style.backgroundColor = "red";
+    }
+  }
+  ulEl.style.display = "flex";
+  info.appendChild(ulEl);
```

Comment:
Very nice solution🏅 Is it a good practice to be consistent when writing code so I wouldn't mix `let` and `var`. You could also add some padding to `li` elements and make sure that images have the same width and height. 

Images don't have links to URLs with book covers as per the instruction but you did a good job🙂